sdk/identity/identity/src/msal/utils.ts

```diff
@@ -34,7 +34,7 @@ const LatestAuthenticationRecordVersion = "1.0";
  */
 export function ensureValidMsalToken(
   scopes: string | string[],
-  msalToken?: MsalToken,
+  msalToken?: MsalToken | null,
```

Comment:
why do we need to change the type to add a null here? if it's already optional.

Reply:
Thanks for asking! It's because MSAL speaks in `null`s - if you look at their signatures a lot of stuff returns `Promise<aThing | null>` and since this function already handles the null case it felt better than to have a bunch of `somethingThatCanBeNull ?? undefined` to convert it to undefined

But, I will call out that this is 100% just my lazyness for this draft and does not have to be part of the final commit 👍 

I am trying to think of better ways to standardize and avoid having something be value, or null, or undefined and we can do that separately